.github/workflows/static-analysis.yml

```diff
@@ -12,8 +12,12 @@ on:
       - "v*.*"
       - "master"
       - "feature/*"
-    tags:
-      - "*"
+  workflow_call:
+    inputs:
+      ref:
+        description: "The git reference to check"
```

Comment:
Nit: in this case, I think "git ref" is the more readable name, even though it's an abbreviation.